__tests__/app.test.js

```diff
@@ -154,6 +155,51 @@ describe("GET /api/articles/:article_id/comments", () => {
     });
 })
 
+describe("POST /api/articles/:article_id/comments", () => {
+    test("status 201, responds with the newly added comment", () => {
+        return request(app)
+        .post("/api/articles/3/comments")
+        .send({
+            username: "icellusedkars",
+            body: "Not my cup of tea"
+        })
+        .expect(201)
+        .then((response) => {
+            expect(response.body.comment).toEqual({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: "icellusedkars",
+                body: "Not my cup of tea",
+                article_id: 3
+            });
+        });
+    });
+    test("status 404, responds with 404 when username does not exist", () => {
+        return request(app)
+        .post("/api/articles/3/comments")
+        .send({
+            username: "ghostbuster1",
+            body: "who you gonna call"
+        })
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Endpoint Not Found")
```

Comment:
I think this error message doesn't quite describe the error that is happening - as it's nothing to do with the endpoint. Can you change the message to something more fitting?